playground/nextjs-app-router/app/mint/page.tsx

```diff
@@ -0,0 +1,56 @@
+'use client';
```

Comment:
why do we need a separate page for this component?

Reply:
I'm hoping to use this page in an iframe for the earnings mint published at onchainkit.xyz/playground/mint

Reply:
can we add a custom app logo and name here? the keys.coinbase.com popup will say `OnchainKit` with a null logo currently, the earnings team will likely want their own customization

Reply:
you may need to override the `WagmiProvider` since it's controlled by the `config` value there